src/pages/omOss.tsx

```diff
@@ -0,0 +1,253 @@
+                </h2>
+                <p className="text-white text-lg">{t.aboutUs.paragraph3}</p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      {/*
+        Phone version version
+      */}
+      <div className="flex flex-row items-center justify-center lg:hidden">
+        <div className="px-[100px] mt-36 bg-[url('/img/grayKth.jpg')] bg-cover bg-no-repeat bg-center h-[150px]">
+          
+        </div>
+      </div>
+      <div className="flex flex-col px-[50px] mt-[100px] mb-[100px] lg:hidden">
+            <div className="flex flex-row items-center justify-center mb-10">
+              <div className="">
+                <h2 className="text-cerise text-xs font-medium">
+                  {" "}
+                  {t.aboutUs.info1}
+                </h2>
+                <p className="text-white text-xs">{t.aboutUs.paragraph1} </p>
+              </div>
+            </div>
+            <div className="flex flex-row">
+              <div className="">
+                <h2 className="text-cerise text-xs font-medium">
+                  {t.aboutUs.info2}
+                </h2>
+                <p className="text-white text-xs">{t.aboutUs.paragraph2}</p>
+              </div>
+
+              <div className="">
+                <h2 className="text-cerise text-xs  font-medium">
+                  {t.aboutUs.info3}
+                </h2>
+                <p className="text-white text-xs">{t.aboutUs.paragraph3}</p>
+              </div>
+            </div>
+          </div>
+
+      <div className="flex flex-row items-center justify-center px-[400px] hidden lg:block">
+        <div className="px-[100px] mt-[100px] bg-[url('/img/groupPictures/projectGroup.jpg')] bg-cover bg-no-repeat bg-center h-[400px] w-full flex flex-row items-center justify-center">
+          <p className="text-cerise text-8xl">{t.aboutUs.subHeader}</p>
+        </div>
+      </div>
+      <div className="flex flex-row items-center justify-center px-[10px] lg:hidden">
+        <div className="bg-[url('/img/groupPictures/projectGroup.jpg')] bg-cover bg-no-repeat bg-center items-center justify-center h-[200px] ">
+          <p className="text-cerise text-7xl text-center">{t.aboutUs.subHeader}</p>
+        </div>
+        </div>
+      
+      <Team
+        team={prTeam}
+        reverse={false}
+        teamPic="/img/groupPictures/prTeam.jpg"
+        names={prNames}
+        teamRoles={t.aboutUs.prTeamRoles}
+        teamName={t.aboutUs.teamNames[0]}
+      />
+
+      <Team
+        team={saleTeam}
+        reverse={true}
+        teamPic="/img/groupPictures/salesTeam.jpg"
+        names={salesNames}
+        teamRoles={t.aboutUs.salesTeamRoles}
+        teamName={t.aboutUs.teamNames[1]}
+      />
+
+      <Team
+        team={massTeam}
+        reverse={false}
+        teamPic=""
+        names={massNames}
+        teamRoles={t.aboutUs.massTeamRoles}
+        teamName={t.aboutUs.teamNames[2]}
+      />
```

Comment:
Imo the team title should be take 50% of the container when we have no picture of the team together.
<img width="1281" alt="image" src="https://github.com/datasektionen/ddagen/assets/66337956/8ccf7b20-9616-4704-b59d-3b6692f31f08">